src/runtime_src/core/tools/xbmgmt2/flash/xspi.cpp

```diff
@@ -546,12 +546,12 @@ int XSPI_Flasher::xclTestXSpi(int index)
 }
```

Comment:
Copyright needs to be updated. A new line after 3 should be added to reflect AMD copyright. For example:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`

In addition, please remove the authors, there is no need to have this in the code base.